src/relayserver/ReputationManager.ts

```diff
@@ -100,9 +133,36 @@ export class ReputationManager {
       .filter(it => Date.now() - it.timestamp < this.config.abuseTimeWindowMs)
       .reduce((previousValue: number, currentValue: ReputationChange) => previousValue + currentValue.change, 0)
     if (-changeInAbuseWindow >= this.config.abuseReputationChange) {
+      /*
       await this.reputationStoreManager.setAbuseFlag(paymaster)
+       */
+      entry.abuseStartedTs = Date.now()
```

Comment:
this is bad: when loading a batch, all are initialized to "now".
should use block-based reputations (to avoid reading the timestamps of all blocks.
we can "calibrate" the block time at startup, using
```js
async averageBlockTime() {
  latest = await getBlock('latest')
  if ( latest.number<1000 ) return 1; //ganache: no average time..
  older = await getBlock(latest.number-1000)
  blockTimeMs = (latest.timestamp - older.timestamp)
  return blockTimeMs
}
```

Reply:
Well
a) reputation abuse mechanism was defined as time-based and does not require block times to function
b) this PR does not modify the logic inside the mechanism, but disconnects it from persistent storage. 

We can discuss change to the Reputation Mechanism separately, but that will not be a blocker by default.